dvc/repo/experiments/queue/tempdir.py

```diff
@@ -98,3 +98,6 @@ def collect_executor(
         exec_result: ExecutorResult,
     ) -> Dict[str, str]:
         return BaseStashQueue.collect_executor(exp, executor, exec_result)
+
+    def get_running_exps(self, fetch_refs: bool = True) -> Dict[str, Dict]:
+        return super(WorkspaceQueue, self).get_running_exps(fetch_refs)
```

Comment:
```suggestion
```

If you are only calling the `super()` implementation with the same exact args, you don't need to implement the function at all in the subclass